components/Header/Header.tsx

```diff
@@ -176,7 +176,7 @@ const Header: React.FunctionComponent<HeaderProps> = ({
         </View>
       )}
 
-      <View style={{ padding: 10, position: 'absolute', right: 0, alignItems: 'flex-end' }}>
+      <View style={{ display: 'none', padding: 10, position: 'absolute', right: 0, alignItems: 'flex-end' }}>
```

Comment:
There are two reasons for this:
1. to put something in the right side of the header to compensate with the left side.
2. we need the user can see the version of the App in all the screens, for screenshots & if we ask the user while using the App & something blocked or an error come up.

If the pixels/orientation are terrible here... I suppose everybody can have an opinion... you can remove those. But IMO, the version & maybe the name of the App are really useful.

This can be a good option, for instance:
Zingo! Test -> `translate('zingo')`
v0.1.1 (119) -> `translate('version')`

Like this:
`<Text style={{ fontSize: 8, color: colors.border }}>{translate('zingo')}</Text>`
`<Text style={{ fontSize: 8, color: colors.border }}>{translate('version')}</Text>`

Reply:
good points

Reply:
@fluidvanadium 

You know what... the only info that is useful here is: the name of the App & the version. (we need this info in all screens because we need that the user can tell us this info if something happen...)
You can remove safely the pixels & the orientation...

If you do those, I approve this PR in 3 secs.